level-2/index.html

```diff
@@ -195,8 +194,9 @@ <h4>
                 </li>
                 <li>Present the user with a choice of one or more <a>share
                 targets</a>, selected at the user agent's discretion. The user
-                MUST be given the option to cancel rather than choosing any of
-                the share targets. Wait for the user's choice.
+                agent MAY display options's message to the user. The user MUST
+                be given the latitude to cancel rather than choosing any of the
```

Comment:
"latitude" seems like a strange word choice here (I guess you're trying to avoid the word "option"). Can you just use "choice"?